### PR TITLE
Translate README.md into KR

### PR DESCRIPTION
**Pull request by unki11**
Translate README.md 기여하기 (Contributing) 부분 한글화

### CODE REVIEW

**Comment by ianychoi** [comment]
Thank you for the PR but I can now say that this is intentional because README.md file can be seen also to non-Korean readers. Maybe for this, keeping both English and Korean parts would be better - would you revise it?

**Comment by unki11** [comment]
I think the same. thank you

File: README.md [hunk 1]
```diff
@@ -23,19 +23,13 @@ Azure SDK는 개발자가 다양한 Azure 서비스를 원하는 프로그래밍
 
 본 저장소는 Microsoft 오픈 소스에 기여하기 위한 가이드라인을 따르고 있습니다.
```
Comment (ianychoi) [inline README.md:25]: Then would you please add one more statement on this part like the below example?

`Note that below is Korean translation of contributing part on [Azure SDK's README.md](https://github.com/Azure/azure-sdk/blob/main/README.md#contributing).`
`아래는 [Azure SDK 저장소에 언급된 기여하기 파트](https://github.com/Azure/azure-sdk/blob/main/README.md#contributing)를 한글로 번역한 것입니다. 참고하셨으면 합니다.`

File: README.md [hunk 2]
```diff
@@ -23,19 +23,13 @@ Azure SDK는 개발자가 다양한 Azure 서비스를 원하는 프로그래밍
...
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.microsoft.com.
+이 프로젝트는 기여 및 제안을 환영합니다. 대부분의 기여는 귀하가 기여를 사용할 권리를 가지고 있으며 실제로 그러한 권리를 당사에 부여한다고 선언하는 기부자 라이센스 계약(CLA)에 동의하기를 요구합니다. 자세한 내용은 https://cla.microsoft.com 을 방문하시면 됩니다.
```
Comment (ianychoi) [inline README.md:26]: Please use a better word than "귀하".

Comment (ianychoi) [reply README.md:26]: Also translation of contributions and contributor needs to be concise - so, please change "기부자".

File: README.md [hunk 3]
```diff
@@ -23,19 +23,13 @@ Azure SDK는 개발자가 다양한 Azure 서비스를 원하는 프로그래밍
...
 
-When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
+pull request를 제출하면 CLA 봇이 자동으로 CLA를 제공하고 PR을 적절하게 꾸밀 필요가 있는지 여부를 결정합니다(예: 레이블, 댓글). 봇이 제공하는 지침을 따르면 됩니다. 이 작업은 CLA를 사용하여 모든 저장소에서 한 번만 수행하면 됩니다.
```
Comment (ianychoi) [inline README.md:28]: In English, first character is usually be capitalized. So, please change "pull" to "Pull".

**Review by ianychoi — CHANGES_REQUESTED** [review]
Thanks for your comment - would you reflect the changes?

**Comment by unki11** [comment]
@microsoft-github-policy-service agree

**Comment by unki11** [comment]
@ianychoi 리뷰 감사합니다. 모든 수정사항 반영했습니다.

**Comment by ianychoi** [comment]
> Author

Almost done! I see that this comment has not been addressed - would you double-check?

"Also translation of contributions and contributor needs to be concise - so, please change "기부자"."

File: README.md [hunk 4]
```diff
@@ -23,9 +23,12 @@ Azure SDK는 개발자가 다양한 Azure 서비스를 원하는 프로그래밍
 
 본 저장소는 Microsoft 오픈 소스에 기여하기 위한 가이드라인을 따르고 있습니다.
 
-이 프로젝트는 기여 및 제안을 환영합니다. 대부분의 기여는 귀하가 기여를 사용할 권리를 가지고 있으며 실제로 그러한 권리를 당사에 부여한다고 선언하는 기부자 라이센스 계약(CLA)에 동의하기를 요구합니다. 자세한 내용은 https://cla.microsoft.com 을 방문하시면 됩니다.
+Note that below is Korean translation of contributing part on [Azure SDK's README.md](https://github.com/Azure/azure-sdk/blob/main/README.md#contributing).    
+아래는 [Azure SDK 저장소에 언급된 기여하기 파트](https://github.com/Azure/azure-sdk/blob/main/README.md#contributing)를 한글로 번역한 것입니다. 참고하셨으면 합니다.
 
-pull request를 제출하면 CLA 봇이 자동으로 CLA를 제공하고 PR을 적절하게 꾸밀 필요가 있는지 여부를 결정합니다(예: 레이블, 댓글). 봇이 제공하는 지침을 따르면 됩니다. 이 작업은 CLA를 사용하여 모든 저장소에서 한 번만 수행하면 됩니다.
+이 프로젝트는 기여 및 제안을 환영합니다. 대부분의 기여는 기부자가 기여를 사용할 권리를 가지고 있으며 실제로 그러한 권리를 당사에 부여한다고 선언하는 기부자 라이센스 계약(CLA)에 동의하기를 요구합니다. 자세한 내용은 https://cla.microsoft.com 을 방문하시면 됩니다.
```
Comment (unki11) [inline README.md:29]: Thanks for your review, but it seems to me that it has changed. please check it?

Comment (ianychoi) [reply README.md:29]: Please be consistent: 기여 & 기부자

File: README.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ Azure SDK는 개발자가 다양한 Azure 서비스를 원하는 프로그래밍
 Note that below is Korean translation of contributing part on [Azure SDK's README.md](https://github.com/Azure/azure-sdk/blob/main/README.md#contributing).    
 아래는 [Azure SDK 저장소에 언급된 기여하기 파트](https://github.com/Azure/azure-sdk/blob/main/README.md#contributing)를 한글로 번역한 것입니다. 참고하셨으면 합니다.
 
-이 프로젝트는 기여 및 제안을 환영합니다. 대부분의 기여는 기부자가 기여를 사용할 권리를 가지고 있으며 실제로 그러한 권리를 당사에 부여한다고 선언하는 기부자 라이센스 계약(CLA)에 동의하기를 요구합니다. 자세한 내용은 https://cla.microsoft.com 을 방문하시면 됩니다.
+이 프로젝트는 기여 및 제안을 환영합니다. 대부분의 기여는 기여자가 기여를 사용할 권리를 가지고 있으며 실제로 그러한 권리를 당사에 부여한다고 선언하는 기여자 라이센스 계약(CLA)에 동의하기를 요구합니다. 자세한 내용은 https://cla.microsoft.com 을 방문하시면 됩니다.
```
Comment (unki11) [inline README.md:29]: 수정 완료했습니다. 친절하게 설명해주셔서 감사합니다.

**Comment by ianychoi** [comment]
Thank you! Approving now

**Pull request merged**